src/components/app/details/appDetails/AppMetrics.tsx

```diff
@@ -39,6 +39,7 @@ export const AppMetrics: React.FC<{ appName: string, environment, podMap: Map<st
     const { appId, envId } = useParams<AppDetailsPathParams>();
     const [calendarValue, setCalendarValue] = useState('');
     const [statusCode, setStatusCode] = useState<StatusTypes>(StatusType.Throughput);
+    const [latency, setLatency] = useState<number>(.999);
```

Comment:
We should move .999 to some constant (DEFAULT_LATENCY) and use it everywhere instead of hardcoding.

Reply:
now it is being used at only one place that is here, so no need to move it out